server/src/configs/config.example.json

```diff
@@ -66,11 +69,11 @@
         "arScanColumn": false,
         "useFor": ["session", "user", "nest", "portal"]
       }
-    }  
+    }
   },
-  "alwaysEnabledPerms": [],
+  "alwaysEnabledPerms": ["map","gyms","pokestops"],
+  "enabledAuthMethods": [],
```

Comment:
Do you mean combining them in `enabledAuthMethods` then in 
`rootRooter.js` and still use that down the way?

Reply:
I believe we are on the same page. You can use enabledAuthMethods beyond, just would prefer them to be combined in `rootRouter.js` because having separate `enabled` keys is clearer for the average map admin.